duckduckgo/iOS
# update content blocking js

Reviewer:
Asana: https://app.asana.com/0/414235014887631/496656327820861
CC:

**Description**:

Update JS content blocking to use whitelist and new version of ABP parser.

Added some utility stuff to allow rebuilding of that with ease if we need to.


**Steps to test this PR**:

Please repeat these tests on a iOS 9.3 and 10+ device/simulator.

1. Visit evanscycles.com - ensure blocking looks appropriate
1. Visit apple.com and drill in to the shopping area - ensure site looks OK
